core-services/image-mirroring/openshift/mapping_origin_4_10

```diff
@@ -93,11 +93,13 @@ registry.ci.openshift.org/origin/4.10:csi-livenessprobe quay.io/openshift/origin
 registry.ci.openshift.org/origin/4.10:csi-node-driver-registrar quay.io/openshift/origin-csi-node-driver-registrar:4.10 quay.io/openshift/origin-csi-node-driver-registrar:4.10.0 quay.io/openshift/origin-csi-node-driver-registrar:latest
 registry.ci.openshift.org/origin/4.10:csi-snapshot-controller quay.io/openshift/origin-csi-snapshot-controller:4.10 quay.io/openshift/origin-csi-snapshot-controller:4.10.0 quay.io/openshift/origin-csi-snapshot-controller:latest
 registry.ci.openshift.org/origin/4.10:csi-snapshot-validation-webhook quay.io/openshift/origin-csi-snapshot-validation-webhook:4.10 quay.io/openshift/origin-csi-snapshot-validation-webhook:4.10.0 quay.io/openshift/origin-csi-snapshot-validation-webhook:latest
+registry.ci.openshift.org/origin/4.10:deploy quay.io/openshift/origin-deploy:4.10 quay.io/openshift/origin-deploy:4.10.0 quay.io/openshift/origin-deploy:latest
```

Comment:
hmm, we may want to pick more Cincy-scoped names for these images.

Reply:
I've added `excluded_images` with a2cc988039 -> e7b890bfc1, so now this doesn't add any mirroring entries.